packetWin7/NPFInstall/NPFInstall/NPFInstall.cpp

```diff
@@ -72,39 +72,39 @@ BOOL PacketInstallDriver40();
 BOOL PacketStopDriver40();
 
 #define STR_COMMAND_USAGE \
-_T("NPFInstall for Npcap ") _T(WINPCAP_VER_STRING) _T(" ( http://npcap.org )\n")\
```

Comment:
Does this change do anything other than add a space before the backslash escape at the end of the line?

If so, is it necessary to support newer versions of MSVC?